dashboard/src/components/Monitor.tsx

```diff
@@ -173,7 +173,10 @@ function DetailBlocks({ chain }: { chain: string }) {
       cancelled = true;
     };
   }, [chain, fromId, currentNetwork]);
-  const rawMessages = messagesWrapper.data;
+  let rawMessages = messagesWrapper.data;
+  if (rawMessages && rawMessages.length === 0) {
+    rawMessages = null;
+  }
```

Comment:
This works. I might have done
```suggestion
  const rawMessages = messagesWrapper.data?.length === 0 ? null : messagesWrapper.data;
```
because :shrug:

Reply:
Understood.